<launch>
  <arg name="model" default="$(env TURTLEBOT3_MODEL)" doc="model type [burger, waffle, waffle_pi]"/>
  <arg name="gui" default="true"/>
  <arg name="paused" default="false"/>
  <arg name="use_sim_time" default="true"/>

  <!-- startup simulated world -->
  <include file="$(find gazebo_ros)/launch/empty_world.launch">
    <arg name="world_name" value="$(find tom_and_jerry_project)/worlds/turtlebot3_empty.world"/>
    <arg name="paused" value="$(arg paused)"/>
    <arg name="gui" value="$(arg gui)"/>
    <arg name="use_sim_time" value="$(arg use_sim_time)"/>
    <arg name="headless" value="false"/>
    <arg name="debug" value="false"/>
  </include>

  <include file="$(find tom_and_jerry_project)/launch/robots.launch"/>

  <!-- run world restting node -->
  <node name="reset_world_q_learning" pkg="tom_and_jerry_project" type="reset_world.py" respawn="true" />

  <!-- run q matrix scripts -->
  <node name="state" pkg="tom_and_jerry_project" type="state.py" />
  
</launch>